src/tokens.ts

```diff
@@ -821,3 +833,35 @@ export interface ILogMessage {
    */
   message: string;
 }
+
+/**
+ * The command IDs used in the git context menus.
+ */
+export enum ContextCommandIDs {
```

Comment:
Enums need to be in a separate file due to a bug in Typescript which causes them to go awry in preference of cyclic imports, birth they do not need to be in tokens if you prefer them elsewhere

Reply:
tokens.ts is fine for me.